CopyQ.py

```diff
@@ -0,0 +1,129 @@
+""" CopyQ Clipboard Management """
+
+import subprocess
+from albertv0 import *
+from shutil import which
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "CopyQ"
+__version__ = "1.0"
+__trigger__ = "cp "
+__author__ = "Benedict Dudel"
+__dependencies__ = ["copyq"]
+
+if which("copyq") is None:
+    raise Exception("'copyq' is not in $PATH.")
+
+iconPath = iconLookup('copyq')
+
+def handleQuery(query):
+    if query.isTriggered:
+        if not query.string or query.string == "":
+            print("show last items")
```

Comment:
remove print. there is not stdout (conceptually) use the [logging functions](https://albertlauncher.github.io/docs/extensions/python/#free-functions) for that